packages/kas/src/__tests__/compilation_test.js

```diff
@@ -0,0 +1,100 @@
+import _ from "underscore";
+
+import * as KAS from "../index.js";
+
+expect.extend({
+    toCompileAs(
+        input: string,
+        expected: number,
+        vars?: {[string]: string | number} = {},
```

Comment:
`vars` can also contain `function` as values, right?

Reply:
Looking at the test cases it looks like the values are only numbers.  FYI the flow types aren't checking anything yet because I didn't turn on flow in these files yet.

Reply:
I didn't turn on flow in these files.  I was going to do that but then it complained about how none of the custom matchers show up in the libdef for `jest`. 😞